src/fs.js

```diff
@@ -45,7 +51,23 @@ function makeDirs(dirs) {
 	});
 }
 
+/** Delete files from a given path */
+function deleteFiles(dirs, options = {}) {
+	dirs = castArray(dirs);
+
+	dirs.forEach(dir => {
+		const hasFile = exists(path.resolve(dir));
```

Comment:
This won’t work with globs. But `del.sync` returns an array of deleted paths so we could show that. And that would allows us to get rid of `forEach` as well ;-)